juniper_official/Chassis/check-failover-configured.rule

```diff
@@ -0,0 +1,71 @@
+/*
+ * Checks if graceful-switchover is configured on a dual RE chassis or not
+ *
+ * First checks if other routing engine is present or not. If present, checks
+ * if failover (set chassis redundancy graceful-switchover) is configured or not.
+ * 
+ * In dual RE chassis, for high availability it is recommended to have faiover
+ * configured. A yellow color is reported if this configuration is not found
+ * in a dual Routing Engine chassis.
+ */
+iceberg {
+    topic chassis.gres {
+        rule check-failover-configured {
+            synopsis "Check whether graceful-switchover is configured or not";
+            description "Checks if other RE is present and if GRES/failover is configured or not";
+            sensor gres-sensor {
```

Comment:
Provide sensor description and synopsis detailing sensor info.
e.g.  synopsis "Routing engine iAgent sensor definition";
        description "Netconf rpc get-route-engine-information to collect data from network device";

Reply:
Addressed